.github/workflows/deploy-testnet.yml

```diff
@@ -231,6 +231,10 @@ jobs:
         scp deploy/testnet/package-lock.json ${{secrets[matrix.user_key]}}@${{secrets[matrix.host_key]}}:/home/${{ secrets[matrix.user_key] }}/atleta/package-lock.json
         scp deploy/config.env ${{secrets[matrix.user_key]}}@${{secrets[matrix.host_key]}}:/home/${{ secrets[matrix.user_key] }}/atleta/config.env
         scp chainspecs/chain-spec.testnet.json ${{secrets[matrix.user_key]}}@${{secrets[matrix.host_key]}}:/home/${{ secrets[matrix.user_key] }}/atleta/chainspec.json
+        ssh -o StrictHostKeyChecking=no ${{secrets.VALIDATOR_16_USER}}@${{secrets.VALIDATOR_16_HOST}} "mkdir -p /home/${{ secrets.VALIDATOR_16_USER }}/atleta/backup_script"
```

Comment:
`${{ var }}`. Without separators,  these lines are hard to read.

Reply:
Fixed